Makefile

```diff
@@ -15,13 +15,18 @@
 
 PROTO_SRC_DIR = ./protos
 PROTO_DST_DIR = ./remme/protos
+RELEASE_NUMBER ?= $(shell git describe --abbrev=0 --tags)
 
-TOTAL_SUPPLY ?= 10000000000000
-export TOTAL_SUPPLY
+include .env
 
-run_dev:
+.PHONY: release
+
+run_dev_no_genesis:
```

Comment:
Production?

Reply:
No, for production use we will release containers on Docker Hub and separate releases in zip-files.